fabfile.py

```diff
@@ -11,12 +12,15 @@
 def prepare():
     local("git pull origin master")
 
-def get_ut():
+def get_ut(input_file,output_file):
+    if input_file == '' or output_file == '':
+        sys.exit("Input file and output file are required.")
+
     put("bin/fetch.pl", "openapc/fetch.pl")
-    put("data/apc_de.csv", "openapc/apc_de.csv")
+    put("data/" + input_file, "openapc/" + input_file)
```

Comment:
This will not work if the supplied argument is a path instead of a file, so something like
`fab -R pub get_ut:data/unibi/unibi_2015-enriched.csv,python/ut_out.csv`
is not possible.